pkg/util/git.go

```diff
@@ -82,14 +81,16 @@ func LatestTagCommitHash(repo *git.Repository) (*string, *plumbing.Hash, error)
 
 	err = gitLog.ForEach(func(c *object.Commit) error {
 		if v, ok := tagIndex[c.Hash.String()]; ok {
-			// logrus.Warnf("Found tag %s; break", v)
 			latestVersionTag = v
 			latestVersionHash = c.Hash
 			return storer.ErrStop
 		}
 
 		if len(c.ParentHashes) > 1 {
-			return errors.New("bff only works with linear history")
+			_, err := GetLatestParentCommit(c)
```

Comment:
hmm is this branch a no-op now?

Reply:
ah correct! Removing the whole if block now.